starter-code/scripts/articleView.js

```diff
@@ -43,6 +46,7 @@ articleView.handleCategoryFilter = function() {
     $('#author-filter').val('');
   });
 };
+})
```

Comment:
In order to get the stretch goal, you would need to fill in the Handlebars templates for the category and author filters with information, compile the templates, and append them to the page.  One way to approach this:
```
Article.prototype.authorsToHtml = function() {
    var sourceHtml = $('#author-filter-template').html();
    var template = Handlebars.compile(sourceHtml);
    return template(this);
 };
```

You would make another method for the categories and do something similar.  Then, you would append the result of calling these methods on each article to the page, similar to how we populated the #articles section with compiled article templates:

articles.forEach(function(a) {
     $('#author-filter').append(a.authorsToHtml());
    $('#category-filter').append(a.categoriesToHtml());
});